test/operators.js

```diff
@@ -42,7 +49,46 @@ describe("Operators tests", function () {
     });
   });
 
-  describe("condition operator", function () {
+  describe("% ** += -= ++ --", function () {
+    it("Should return c % b", function () {
+      assert.strictEqual(c % b, 1);
+    });
+    it("Should return b ** a", function () {
+      assert.strictEqual(b ** a, 9);
+    });
+    it("Should return a += b", function () {
+      assert.strictEqual((a += b), 5);
+    });
+    it("Should return a -= c", function () {
+      assert.strictEqual((a -= c), -5);
+    });
+    it("Postfix++, asign x to y and then x increments by 1", function () {
+      let x = 100;
+      const y = x++;
+      assert.strictEqual(x, 101);
+      assert.strictEqual(y, 100);
+    });
+    it("Prefix++, x increments by 1 and then assign to y", function () {
+      let x = 100;
+      const y = ++x;
+      assert.strictEqual(x, 101);
+      assert.strictEqual(y, 101);
+    });
```

Comment:
These are exellent test cases to demonstrate how `++` works. :clap: Same with `--` below. :tada: